example/src/AppBar/AppBarPage.tsx

```diff
@@ -13,9 +14,21 @@ export const AppBarPage = () => {
         { title: 'option 4', onPress: () => console.log('option') },
     ];
 
+    const options: DropDownOption[] = [
+        { value: 'Revalorisation auto' },
+        { value: 'Option 2' },
+        { value: 'Option 3' },
+        { value: 'Option 4' },
+    ];
+    const [selected, setSelected] = useState<DropDownOption>();
+
+    useEffect(() => {
+        setSelected(options[0]);
+    }, []);
+
     return (
-        <View>
-            <AppBar onPress={() => {}} text={'Title'}>
+        <View style={{ backgroundColor: 'white', flex: 1 }}>
```

Comment:
Nitpick, but could you use the `Screen` component instead ?
It already has the colors and stuff setup